client/blocks/product-purchase-features/product-purchase-features-list/monetize-site.jsx

```diff
@@ -21,7 +21,7 @@ export default localize( ( { selectedSite, translate } ) => {
 						'WordAds lets you earn money by displaying promotional content.'
 					)
 				}
-				buttonText={ translate( 'Start Earning' ) }
+				buttonText={ translate( 'Start earning' ) }
```

Comment:
Warning: 19 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Earnings' )`&mdash; translations: **21**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)